src/Components/ExternalResult/FacilitiesSelectDialogue.tsx

```diff
@@ -1,91 +1,39 @@
-import {
-  Button,
-  Dialog,
-  DialogActions,
-  DialogContent,
-  DialogTitle,
-} from "@material-ui/core";
-import CheckCircleOutlineIcon from "@material-ui/icons/CheckCircleOutline";
 import React from "react";
+import ConfirmDialogV2 from "../Common/ConfirmDialogV2";
 import { FacilitySelect } from "../Common/FacilitySelect";
 import { FacilityModel } from "../Facility/models";
-import { makeStyles } from "@material-ui/core/styles";
-
-const useStyles = makeStyles({
-  paperFullWidth: {
-    overflowY: "visible",
-  },
-  dialogContentRoot: {
-    overflowY: "visible",
-  },
-});
+import { FieldLabel } from "../Form/FormFields/FormField";
 
 interface Props {
+  show: boolean;
   handleOk: () => void;
   handleCancel: () => void;
   selectedFacility: FacilityModel;
   setSelected: (e: any) => void;
 }
 
 const FacilitiesSelectDialog = (props: Props) => {
-  const { handleOk, handleCancel, selectedFacility, setSelected } = props;
-  const classes = useStyles();
-
-  const handleEscKeyPress = (event: any) => {
-    if (event.key === "Escape") {
-      handleCancel();
-    }
-  };
+  const { show, handleOk, handleCancel, selectedFacility, setSelected } = props;
 
   return (
-    <Dialog
-      open={true}
-      fullWidth={true}
-      classes={{
-        paperFullWidth: classes.paperFullWidth,
-      }}
-      onKeyDown={(e) => handleEscKeyPress(e)}
+    <ConfirmDialogV2
+      action={"Continue"}
+      title={<FieldLabel className="text-lg">Search for Facility</FieldLabel>}
+      show={show}
+      variant={"primary"}
+      disabled={!selectedFacility}
+      onClose={handleCancel}
+      onConfirm={handleOk}
     >
```

Comment:
In the issue, the confirm dialog was meant for operations that needs explicit approval. Like delete, archive, etc. And not meant to be used for simple operations like selecting a facility.